docs/my-v2-addon/package.json

```diff
@@ -65,6 +65,7 @@
   },
   "dependencies": {
     "@embroider/addon-shim": "^1.10.0",
+    "@glimmer/component": "^2.0.0",
```

Comment:
If we keep `@glimmer/component` in `devDependencies`, we get errors like the following:

```sh
Could not find module `@glimmer/component` imported from `my-v2-addon/components/ui/page/subsection`
```